doc/development/02devcontainer.md

```diff
@@ -0,0 +1,30 @@
+# Devcontainer
+
+We are also supporting a containerized setup(docker/podman). For more information checkout the README inside [.devcontainer](https://github.com/ElektraInitiative/PermaplanT/blob/master/.devcontainer/README.md).
```

Comment:
```suggestion
We are also supporting a containerized setup(docker/podman). For more information checkout the README inside [.devcontainer](https://blob.permaplant.net/master/.devcontainer/README.md).
```

Reply:
Generally you can but it in many cases (e.g. links to outside of doc/) it won't work on doc.permaplant.net etc.